docs/tutorials/check_imbalance.md

```diff
@@ -0,0 +1,491 @@
+---
```

Comment:
If we want to have it in the main tutorial, I can add 1-2 lines. Otherwise I would not mess with the tutorial once it is getting into shape. This notebook is rather for internal use and I would remove it before merging.

Reply:
:+1:

Atm, I have a section "experimental plots" at the bottom of the tutorial with plotting functions that are not as thoroughly tested yet and/or we still need to wrap them into the storyline of a tutorial. 
I think it's fine to add a small example there. 

At some later point, we can consider splitting up tutorials by specific analysis use-cases.

Reply:
Ok, I'll add the volcano there then.